src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -0,0 +1,233 @@
+{
+  "meta": {
+    "releasedOn": "2023.06.08"
+  },
+  "added": [
+    {
+      "what": [
+        "option for transparent color in the color picker."
+      ],
+      "why": [
+        "to give users more configuration options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1211"
+      ]
+    },
+    {
+      "what": [
+        "the option to set a record list filter as a preset filter for a user."
```

Comment:
Was this for a user or for a role? iirc it was for a role but I could be wrong here

Reply:
it's for a user